### PR TITLE
Post_PR657: Add workaround comment

### PR DESCRIPTION
**Pull request by zjing14**
- Add a comment following workaround rules for tracking the hotfix (PR #657) of issue #659

### CODE REVIEW

File: src/solver/conv_hip_implicit_gemm_nonxdlops_common.cpp [hunk 1]
```diff
@@ -93,9 +93,13 @@ bool PerformanceImplicitGemm::IsValid(const ConvolutionContext& ctx) const
     }
 
     // divide block work by [K, B]
-    if(!(K % KPerBlock == 0 && B % BPerBlock == 0 && E % (2 * EPerBlock) == 0))
+    if(!(K % KPerBlock == 0 && B % BPerBlock == 0 && E % EPerBlock == 0))
         return false; // wrong! cannot divice N evenly among thread
 
+    // WORKAROUND_ISSUE_659
+    if(E % (2 * EPerBlock) != 0)
+        return false;
```
Comment (atamazov) [inline src/solver/conv_hip_implicit_gemm_nonxdlops_common.cpp:101]: Could you please
```suggestion
#if WORKAROUND_ISSUE_659
    if(E % (2 * EPerBlock) != 0)
        return false;
#endif
```
and
```C++
#define WORKAROUND_ISSUE_659 1
```
at the beginning of this file?

**Review by atamazov — CHANGES_REQUESTED** [review]
Thank you, @zjing14, but some minor change is still required.

**Comment by atamazov** [comment]
@zjing14 Please let me know if you don't have time for this, and I will do.

File: src/solver/conv_hip_implicit_gemm_nonxdlops_common.cpp (hunk 1 above)
Comment (zjing14) [reply src/solver/conv_hip_implicit_gemm_nonxdlops_common.cpp]: Pushed. Sorry for forgetting this.

Comment (atamazov) [reply src/solver/conv_hip_implicit_gemm_nonxdlops_common.cpp]: NP!

**Review by atamazov — APPROVED** [review]
LGTM!

**Pull request merged**